features/bounding-box-zoeken.feature

```diff
@@ -0,0 +1,131 @@
+        | geometriecoördinaten                                                                                                        |
+        | [ [118315.606,404844.967],[118324.183,404843.509],[118325.179,404849.365],[118316.604,404850.835],[118315.606,404844.967] ] |
+        Als met "GET" "/panden?bbox=118310.304,404850.835,118320.604,404860.592" wordt gezocht
+        Dan bevat het resultaat het pand met identificatie "0826100000036343"
+
+    Scenario: bbox coördinaat ligt op de verbindingslijn tussen twee coördinaten van pand
+        Gegeven het pand met identificatie "0826100000036343" heeft de volgende geometrie polygoon coördinaten
+        | geometriecoördinaten                                                                                                        |
+        | [ [118315.606,404844.967],[118324.183,404843.509],[118325.179,404849.365],[118316.604,404850.835],[118315.606,404844.967] ] |
+        Als met "GET" "/panden?bbox=118324.681,404846.437,118330.911,404840.524" wordt gezocht
+        Dan bevat het resultaat het pand met identificatie "0826100000036343"
+
+    Scenario: een bbox coördinaat komt overeen met een geometrie coördinaat van pand
+        Gegeven het pand met identificatie "0826100000036343" heeft de volgende geometrie polygoon coördinaten
+        | geometriecoördinaten                                                                                                        |
+        | [ [118315.606,404844.967],[118324.183,404843.509],[118325.179,404849.365],[118316.604,404850.835],[118315.606,404844.967] ] |
+        Als met "GET" "/panden?bbox=118324.183,404843.509,118355.628,404839.365" wordt gezocht
+        Dan bevat het resultaat het pand met identificatie "0826100000036343"
+
+    Scenario: een bbox coördinaat komt overeen met puntgeometrie verblijfsobject
+        Gegeven het adresseerbaar object met identificatie "0226010000038820" heeft de volgende geometrie punt coördinaten
+        | geometriecoördinaten    |
+        | [196733.427,439931.991] |
+        Als met "GET" "/adresseerbareobjecten?bbox=196713.427,439911.991,196733.427,439931.991" wordt gezocht
+        Dan bevat het resultaat het adresseerbaar object met identificatie "0226010000038820"
+
+    Scenario: puntgeometrie van een verblijfsobject ligt op een verbindingslijn van de bbox 
+        Gegeven het adresseerbaar object met identificatie "0226010000038820" heeft de volgende geometrie punt coördinaten
+        | geometriecoördinaten    |
+        | [196733.427,439931.991] |
+        Als met "GET" "/adresseerbareobjecten?bbox=196713.427,439911.142,196733.427,439951.935" wordt gezocht
+        Dan bevat het resultaat het adresseerbaar object met identificatie "0226010000038820"
+
+Rule: Resource wordt gevonden wanneer de coördinaat of rand van de bbox-parameter niet meer dan een halve milimeter van de buitenrand van de resourcegeometrie ligt
```

Comment:
Voorstel : 
Rule: Resource wordt gevonden wanneer een coördinaat of de rand van de bbox-parameter niet meer dan een halve milimeter buiten de buitenrand van de resourcegeometrie ligt